src/sentry/static/sentry/app/components/qrcode.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+type Props = {
+  code: (1 | 0)[][];
+};
+
+const Qrcode = ({code}: Props) => (
+  <Table>
+    <tbody>
+      {code.map((row, i) => (
+        <tr key={i}>
+          {row.map((cell, j) => (cell ? <BlackCell key={j} /> : <WhiteCell key={j} />))}
+        </tr>
+      ))}
+    </tbody>
+  </Table>
```

Comment:
One day we should consider changing this into canvas rendering. This is a gob of HTML to make an image.

Reply:
Yeah agreed